js/controllers/todoCtrl.js

```diff
@@ -158,11 +212,11 @@ $scope.doReply = function (todo) {
 	$scope.todos.$save(todo);
 	
 	// TODO: Seems to be superfluous if not trusting the desc as HTML anyway
-	var desc = $scope.XssProtection(newTodo);
```

Comment:
The input is currently in itself protected against http inputs.
Thus this is deprecated. The XssProtection causes issues since some characters end up being replaced and displayed incorrectly.